apps/src/feedback.js

```diff
@@ -807,19 +810,18 @@ FeedbackUtils.prototype.getShowCodeElement_ = function(options) {
       (options.response.total_lines !== numLinesWritten));
   var totalNumLinesWritten = shouldShowTotalLines ? options.response.total_lines : 0;
 
-  showCodeDiv.innerHTML = require('./templates/showCode.html.ejs')({
-    numLinesWritten: numLinesWritten,
-    totalNumLinesWritten: totalNumLinesWritten
+  var generatedCodeElement = this.getGeneratedCodeComponent_({
+    generatedCodeDescription: options.appStrings && options.appStrings.generatedCodeDescription
   });
 
-  var showCodeButton = showCodeDiv.querySelector('#show-code-button');
-  showCodeButton.addEventListener('click', _.bind(function () {
-    var generatedCodeElement = this.getGeneratedCodeElement_({
-      generatedCodeDescription: options.appStrings && options.appStrings.generatedCodeDescription
-    });
-    showCodeDiv.appendChild(generatedCodeElement);
-    showCodeButton.style.display = 'none';
-  }, this));
+  ReactDOM.render(<CodeWritten
+    numLinesWritten = {numLinesWritten}
+    totalNumLinesWritten = {totalNumLinesWritten}
+    generatedCodeElement = {generatedCodeElement}
```

Comment:
Would it make more sense to express this as:
```
<CodeWritten ...>
  {generatedCodeComponent}
</CodeWritten>
```
i.e. passing in the component as a child instead of a prop?